test/scripts/integration_tests.py

```diff
@@ -315,6 +331,30 @@ def __check_metrics(self, events, managed, created, deleted, updated):
 
         raise AssertionError("Failed to see the expected number of events. {}".format(metrics))
 
+    def __check_timestamp(self, metric, timestamp, num_events):
+        """
+        Assert timestamp for metric is greater than timestamp passed in
+        :param metric: name of metric to check
+        :param timestamp: timestamp to compare metric
+        :param num_events: expected number of events
+        """
+        attempts = 10
+        while attempts > 0:
+            metrics_response = requests.get(self.__metrics_url)
+            metrics_response.raise_for_status()
+            metrics = metrics_response.text.split("\n")
+            if "releases_events_total {}".format(num_events) not in metrics:
+                time.sleep(1)
+                attempts -= 1
+            else:
+                for line in metrics:
+                    if re.match(metric, line):
+                        metric_ts = line.split(" ")[1]
+                        self.assertTrue(float(metric_ts) > timestamp)
```

Comment:
If I'm reading the above right we are using seconds for the timestamp, which makes me worry we might be able to run into a case where we do an update/create/delete in the same second that we do the check making the > false flag. Possibly adjusting this to >= covers that and doesn't have any negative effects?

Reply:
@drush211 The metric time uses nanosecond precision and python's time uses microsecond precision. I don't think this will be an issue, but I'm happy to update it.

Reply:
I'm good with this then. Just wanted to double check.